spacy_llm/backends/integration/langchain.py

```diff
@@ -55,14 +56,10 @@ def backend_langchain(
     type_to_cls_dict: Dict[str, Type[BaseLLM]] = langchain.llms.type_to_cls_dict
 
     if api in type_to_cls_dict:
-        backend = type_to_cls_dict[api](**config)
-
-        query_fn = query_langchain() if query is None else query
-
-        def _query(prompts: Iterable[Any]) -> Iterable[Any]:
-            return query_fn(backend, prompts)
-
-        return _query
+        return Backend["BaseLLM", Any, Any](
```

Comment:
Ah, if we end up using this often then maybe it defeats the purpose of `TypeVar`s?

Reply:
Outdated, see https://github.com/explosion/spacy-llm/pull/117#discussion_r1205157647.